src/Nethermind/Nethermind.Optimism/Rpc/OptimismEthRpcModule.cs

```diff
@@ -0,0 +1,285 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Nethermind.Blockchain.Find;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Crypto;
+using Nethermind.Facade;
+using Nethermind.Facade.Eth;
+using Nethermind.Facade.Filters;
+using Nethermind.Int256;
+using Nethermind.JsonRpc;
+using Nethermind.JsonRpc.Client;
+using Nethermind.JsonRpc.Data;
+using Nethermind.JsonRpc.Modules.Eth;
+using Nethermind.Serialization.Rlp;
+using Nethermind.State.Proofs;
+using Nethermind.TxPool;
+
+namespace Nethermind.Optimism;
+
+public class OptimismEthRpcModule : IEthRpcModule
```

Comment:
why not just inherit from `EthRpcModule` and override the `sendTransaction` methods?

Reply:
We need to change Receipt type in getTxReceipt, so we need even different interface